src/intrinsics/ecma262/Math.js

```diff
@@ -200,8 +201,18 @@ export default function(realm: Realm): ObjectValue {
 
   // ECMA262 20.2.2.27
   obj.defineNativeMethod("random", 0, context => {
-    if (realm.mathRandomGenerator !== undefined) {
-      return new NumberValue(realm, realm.mathRandomGenerator());
+    let mathRandomGenerator = realm.mathRandomGenerator;
+    if (mathRandomGenerator !== undefined) {
+      let loc = realm.currentLocation;
+      let error = new CompilerDiagnostic(
+        "Result of Math.random() is made deterministic via a fixed mathRandomSeed",
+        loc,
+        "PP0001",
+        "Information"
```

Comment:
This message sound more like a warning than a mere bit of information. I.e. it warns that thing might go wrong because of the code that is being pointed out.

Reply:
However, the user had to actively supply option --mathRandomSeed, so it's something expected, and this just notifies the user how that option was used.